src/schema/v2/conversation/update_conversation_mutation.ts

```diff
@@ -1,34 +1,66 @@
+import { GraphQLBoolean } from "graphql"
 import { GraphQLString, GraphQLNonNull } from "graphql"
 import { mutationWithClientMutationId } from "graphql-relay"
 import Conversation from "schema/v2/conversation"
 import { ResolverContext } from "types/graphql"
 
-export default mutationWithClientMutationId<any, any, ResolverContext>({
+interface UpdateMessageMutationInputProps {
+  conversationId: string
+  fromLastViewedMessageId: string
+}
+
+export default mutationWithClientMutationId<
```

Comment:
I guess this mutation doesnt use the `orError` pattern (and can't bc its a breaking change and there are existing clients?)

Reply:
That might make sense as a code comment in this file maybe.